content/en/02-installing-pixie/03-install-guides/04-production-readiness-shared-cloud.md

```diff
@@ -0,0 +1,130 @@
+---
+title: "Production Readiness - Shared Cloud"
+metaTitle: "Production Readiness - Shared Cloud"
+metaDescription: "Sharing a single Pixie Cloud across multiple Pixie deployments"
+order: 4
+---
+
+Pixie allows you to connect multiple Kubernetes clusters to a single Pixie Cloud instance. The main advantage of such a deployment is that you can monitor all your Kubernetes clusters from a single point. 
+
+This guide explains how to share a Pixie Cloud instance across multiple Pixie deployments.
+
+
+## Prerequisites
+
+- A domain name that you own
+- Prerequisites of the install method your choice; [Self-hosted Pixie](/installing-pixie/install-guides/self-hosted-pixie/#prerequisites) or [Air gapped Pixie](/installing-pixie/install-guides/airgap-pixie/#prerequisites)
+
+## Steps
+
+## 1. Install the NGINX Ingress Controller
+
+Install the NGINX Ingress Controller in your Kubernetes cluster. Thereafter note the IP address assigned to the `ingress-nginx-controller` Load Balancer service. All requests to the Pixie Cloud will be sent through this.
+
+## 2. Create DNS A records
+
+Two DNS `A` records need to be created pointing to the NGINX Ingress controller IP address obtained above.
+
+Suppose that your Pixie custom domain name is `pixie.example.com` and the IP address obtained above is `a.b.c.d`. Two `A` records need to be created as follows.
+```
+pixie.example.com           a.b.c.d
+work.pixie.example.com      a.b.c.d
+```
+
+## 3. Install Pixie Cloud
+
+If you are using the [Self-Hosted installation](/installing-pixie/install-guides/self-hosted-pixie/),
+1. Follow steps 1 - 5 and 8 in [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud).
+2. Comment lines 94 to 106 in `./scripts/create_cloud_secrets.sh`
+3. Execute the script as explained in step 9 of [Deploy Pixie cloud](/installing-pixie/install-guides/self-hosted-pixie/#1.-deploy-pixie-cloud)
```

Comment:
Same here, let's make this `Deploy Pixie Cloud` to keep capitalization consistent.

Reply:
Fixed through https://github.com/pixie-io/docs.px.dev/pull/245/commits/eb0693c49c18d9f219c0275b718d4f3d82dbbf5a